src/solvers/file_solution_history.C

```diff
@@ -40,7 +40,7 @@ void FileSolutionHistory::find_stored_entry(Real time)
   if (stored_solutions.begin() == stored_solutions.end())
     return;
 
-  libmesh_assert (stored_sols != stored_solutions.end());
+  //libmesh_assert (stored_sols != stored_solutions.end());
```

Comment:
What happened here?

Reply:
I dont think we need that assert anymore, now that we are using map::lower_bound.

Reply:
Yeah, we're not relying on stored_sols at all here now.  So take out the assert entirely?  When I see a commented-out assert in code I foolishly interpret that as "we used to be able to guarantee this but now we can't" rather than "we used to require this but now we don't", and thinking our guarantees are getting weaker always makes me antsy.

Reply:
No worries, the idea behind commenting rather than direct removal was to make sure I am not overlooking some other case where that assert might be useful.